apps/studio/src/store/modules/settings/SettingStoreModule.ts

```diff
@@ -54,6 +54,10 @@ const SettingStoreModule: Module<State, any> = {
     menuStyle(state) {
       if (!state.settings.menuStyle) return 'native'
       return state.settings.menuStyle.value
+    },
+    sortOrder(state) {
```

Comment:
I think this adds much needed functionality. But it does mean that connections cannot be dragged / dropped for sorting. Do you think that's a good trade off?